File: Sistemas de Gestión/pythonProjects/store_manager_erp_ignacio/classes/product.py
class Product:
    def __init__(self, id, name, price, stock_quantity):
        self.id = id
        self.name = name
        self.price = price
        self.stock_quantity = stock_quantity
    
    def set_stock_quantity(self, stock_quantity):
        self.stock_quantity = stock_quantity

    def get_price(self):
        return self.price
    
    def to_string(self):
        return f"ID: {self.id}\nNombre: {self.name}\nPrecio: {self.price}\nCantidad en stock: {self.stock_quantity}"

File: Programacion Multimedia/ionic/examenIgnacio/ferreteriaIgnacio/src/models/product.ts
export class Product {
    private _name: string;
    private _price: number;
    private _isInShoppingCart: boolean;
    private _quantity;

    constructor(name: string, price: number) {
        this._name = name;
        this._price = price;
        this._isInShoppingCart = false;
        this._quantity = 0;
    }

    public get name(): string {
        return this._name;
    }
    public set name(value: string) {
        this._name = value;
    }

    public get price(): number {
        return this._price;
    }
    public set price(value: number) {
        this._price = value;
    }

    public get isInShoppingCart(): boolean {
        return this._isInShoppingCart;
    }
    public set isInShoppingCart(value: boolean) {
        this._isInShoppingCart = value;
    }

    public get quantity() {
        return this._quantity;
    }
    public set quantity(value) {
        this._quantity = value;
    }

    
}
